CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ the `CHANGELOG.md` for each individual package for any changes relevant for deve
 If you are looking for the Ledger Releasing and Versioning Process then you can find it in
 [RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd).
 
+## 8.6
+
+- Performance tweaks related to `Umap.size`
```

Comment:
This is not a user facing change, also it had no measurable impact on performance
```suggestion
```